README.md

```diff
@@ -21,19 +21,48 @@ There are a variety of systems that use merkle-tree and hash-chain inspired data
 
 In short: JSON documents with named merkle-links that can be traversed.
 
+## Table of Contents
+
+- [Repositories](#repositories)
+  - [Specifications](#specifications)
+  - [Implementations](#implementations)
+    - [New implementations](#new-implementations)
+  - [Other Tools](#other-tools)
+  - [Meta repos](#meta-repos)
+- [Lead](#lead)
+- [Contribute](#contribute)
+- [License](#license)
+
 ## Repositories
 
-IPLD is a multifaceted, distributed effort. Here are our current repositories:
+IPLD is a multifaceted, distributed effort.
+
+### Specifications
 
+- [specs](https://github.com/ipld/specs) - The specifications for IPLD
 - [cid](https://github.com/ipld/cid) - Self-describing content-addressed identifiers for distributed systems
+
+### Implementations
+
+- [**JavaScript:** js-ipld-dag-cbor](https://github.com/ipld/js-ipld-dag-cbor) - JavaScript implementation of the IPLD spec.
+- [**Go:** ipfs/go-ipld](https://github.com/ipfs/go-ipld) - The Go implementation of IPLD.
```

Comment:
Note that go-ipld isn't the IPLD implementation, in fact, it is almost out of date for 1 year

![image](https://cloud.githubusercontent.com/assets/1211152/21292265/a2580318-c4f6-11e6-8ec9-3d742c6fd886.png)

The implementations are:

- https://github.com/ipfs/go-ipld-btc
- https://github.com/ipfs/go-ipld-zcash
- https://github.com/ipfs/go-ipld-cbor
- https://github.com/ipfs/go-ipld-git
- https://github.com/ipfs/go-ipld-node

//cc @whyrusleeping 

@RichardLitt also read https://github.com/ipld/interface-ipld-format#background